lib/linter.coffee

```diff
@@ -75,7 +80,7 @@ class Linter
         if rubocopYmlPath
           @copyFile rubocopYmlPath, path.join(tempDir, '.rubocop.yml')
       .then =>
-        @findHamlLintYmlFile filePath
+        @findHamlLintYmlFile(projectPath)
```

Comment:
No need for this, `helpers.find()` already searches all parent directories (to the root of the file system), which would include this. The only real change this would do is prevent a `.haml-lint.yml` file in the same directory as the current file from working, assuming it was a sub-directory of the full project.